stylesheets/scss/includes/_typography.scss

```diff
@@ -0,0 +1,34 @@
+@mixin headingtype($type: heading1) {
```

Comment:
@reubenmoes I'll add comments describing what needs to be done to add it and how we can go about using it.